utils/context/AuthProvider.js

```diff
@@ -74,28 +73,26 @@ export function AuthProvider({ children }) {
           location: accountInfoObj.location
         }
         await axios.post(`http://localhost:3000/api/users`, postBody)
```

Comment:
Probably don't need to use async await here since we didn't really solve the bug. Lets just keep code clean until we tackle it